pycc/ccresponse.py

```diff
@@ -218,8 +292,6 @@ def linresp(self, A, B, omega, e_conv=1e-13, r_conv=1e-13, maxiter=200, max_diis
                     X1[X_key], X2[X_key], polar = self.solve_right(self.pertbar[pertkey], -omega, e_conv, r_conv, maxiter, max_diis, start_diis)
                     check.append(polar)
 
-        return check
```

Comment:
don't want to return the `check` dictionary anymore?

Reply:
Not within the `linresp` function; that was a holdover from my moving all the pseudoresponse checks into the `pertcheck` function.  The `linresp` function will eventually return the linear response function (obviously).